projects/adxrs290-pmdz/src/platform/stm32/platform_src.mk

```diff
@@ -2,13 +2,17 @@ INCS += $(PLATFORM_DRIVERS)/stm32_delay.h     \
         $(PLATFORM_DRIVERS)/stm32_gpio.h      \
         $(PLATFORM_DRIVERS)/stm32_hal.h       \
         $(PLATFORM_DRIVERS)/stm32_spi.h       \
+        $(PLATFORM_DRIVERS)/stm32_pwm.h       \
```

Comment:
is this needed?

Reply:
The adxrs290-pmdz project has the TIM module enabled in the .ioc file. Hence added the PWM drivers here as we are making a check for the HAL_TIM_MODULE_ENABLED in the stm32_spi.c driver